pkg/tsdb/v3iotsdb_integration_test.go

```diff
@@ -199,6 +199,17 @@ func TestQueryData(t *testing.T) {
 			aggregators: "sum",
 			expected:    map[string][]tsdbtest.DataPoint{"sum": {{Time: 1532940510, Value: 701.0}}}},
 
+		{desc: "Should ingest and query an aggregator EXTRA", metricName: "cpu",
```

Comment:
Change the test description + add a todo that says this test should return to buckets as a result, as we discussed today.
Can also just ignore the test with `ignore reason`